src/hotspot/share/jfr/metadata/metadata.xml

```diff
@@ -1081,6 +1081,10 @@
     <Field type="uint" name="stallCount" label="Stall Count" description="The number of Java threads stalled by the GC locker" />
   </Event>
 
+  <Event name="Finalizer" category="Java Application" label="Finalizer" thread="false" startTime="false" period="endChunk">
```

Comment:
Would it make sense for this event to be under the "Java Virtual Machine, GC" category?

Reply:
My argument for placing the event in the Java Application category:

The finalization concept is, or maybe more correctly has been, reified in the Java language. The Java programmer decides to enlist the exposed functionality, similar to Java language Exceptions and Errors. JFR already represents events for these latter concepts in the Java Application category because programmers directly interact with and, in contrast to many events in the JVM category, have more control over them.